molior-deploy

```diff
@@ -638,49 +647,70 @@ include_deploy_conf()
     curl -n -f -s --retry 10 -o $WORK_DIR/Sources.func $baseurl/dists/$dist/main/source/Sources
     lines=`wc -l $WORK_DIR/Sources 2>/dev/null | cut -d' ' -f1`
     if [ -n "$lines" ] && [ "$lines" -eq 0 ]; then
-      # try next baseurl
       continue
     fi
-    srcinfo=`/usr/lib/molior-tools/find-latest-version.pl $deploy_conf < $WORK_DIR/Sources.func`
+    srcinfo=`/usr/lib/molior-tools/find-latest-version.pl $package < $WORK_DIR/Sources.func`
     if [ $? -ne 0 ]; then
-      # try next baseurl
       continue
     fi
-    CONFPKG_NAME=`echo $srcinfo | cut -d ' ' -f 1`/`echo $srcinfo | cut -d ' ' -f 2`
-    CONFPKG_REV=`echo $srcinfo | cut -d ' ' -f 3`
-    if [ -n "$CONFPKG_NAME" ]; then
-        FUNC_SOURCEMIRROR=$baseurl
-      # found package
+    PKG_NAME=`echo $srcinfo | cut -d ' ' -f 1`/`echo $srcinfo | cut -d ' ' -f 2`
+    PKG_REV=`echo $srcinfo | cut -d ' ' -f 3`
+    if [ -n "$PKG_NAME" ]; then
+      FUNC_SOURCEMIRROR=$baseurl
       break
     fi
   done
   rm -f $WORK_DIR/Sources.func
-  if [ -z "$CONFPKG_NAME" ]; then
-    log_error "Error finding '$deploy_conf' source package in apt sources"
+  if [ -z "$PKG_NAME" ]; then
+    log_error "Error finding '$package' source package in apt sources"
   fi
-  confdir=`mktemp -d $WORK_DIR/conf-XXXXX`
-  old_dir=$PWD
-  cd $confdir
-  curl -n -f -s --retry 10 -O -J $FUNC_SOURCEMIRROR/$CONFPKG_NAME
-  exit_on_error "Error downloading $FUNC_SOURCEMIRROR/$CONFPKG_NAME"
 
-  tar xf `basename $CONFPKG_NAME`
-  exit_on_error "Error extracting `basename $CONFPKG_NAME`"
+  pkgdir=`mktemp -d $WORK_DIR/conf-XXXXX`
+  old_dir=$PWD
+  cd $pkgdir
+  curl -n -f -s --retry 10 -O -J $FUNC_SOURCEMIRROR/$PKG_NAME
+  exit_on_error "Error downloading $FUNC_SOURCEMIRROR/$PKG_NAME"
+
+  tar xf `basename $PKG_NAME`
+  exit_on_error "Error extracting `basename $PKG_NAME`"
+  rm -f `basename $PKG_NAME`
+
+  log "sourcing from $package ($PKG_REV): $source_file"
+  
+  # provide new SOURCE_DIR for pkg includes to work (recursively)
+  if [ "$mode" = "deploy_conf" ]; then
+    eval orig_SOURCE_DIR_deploy_conf_$deploy_conf_level=$SOURCE_DIR
+  else
+    eval orig_SOURCE_DIR_file_$include_file_level=$SOURCE_DIR
+  fi
 
-  rm -f `basename $CONFPKG_NAME`
+  SOURCE_DIR=$pkgdir/$package
 
-  log "sourcing from $deploy_conf ($CONFPKG_REV): $source_file"
-  # provide new SOURCE_DIR for conf pkg includes to work (recursively)
-  eval orig_SOURCE_DIR_$deploy_conf_level=$SOURCE_DIR
-  SOURCE_DIR=$confdir/$deploy_conf
-  cd `dirname $deploy_conf/$source_file`
-  . $SOURCE_DIR/$source_file
-  cd - >/dev/null
-  eval SOURCE_DIR=\$orig_SOURCE_DIR_$deploy_conf_level
+  if [ "$mode" = "source" ]; then
```

Comment:
what is "source" ?

Reply:
fixed, changed to the wanted value `deploy_conf`